base/docs/basedocs.jl

```diff
@@ -2281,6 +2282,21 @@ instruction, otherwise it'll use a loop.
 """
 replacefield!
 
+"""
+    setfieldonce!(value, name::Union{Int,Symbol}, desired,
+                  [success_order::Symbol, [fail_order::Symbol=success_order]) -> success::Bool
```

Comment:
What is the default `success_order`?

Reply:
The default is not to have a success_order (in some places, equivalent to `:not_atomic`, but not identical)